src/ImageFitting/ImageFitter.h

```diff
@@ -220,13 +201,13 @@ class ImageFitter {
      * @param initial_values Initial fitting parameter
      * @param offset_x X-axis offset from the fitting region to the entire image
      * @param offset_y Y-axis offset from the fitting region to the entire image
-     * @return A tuple of Gaussian parameters
+     * @return A tuple of Gaussian parameters: center x, center y, amplitude, FWHM x, FWHM y, and position angle
```

Comment:
Add struct to Util.h instead of tuple, use in ImageFitter and InitialValueCalculator

Reply:
Added a new struct `GaussianParams` in `Util.h`. Also moved `GetGaussianComponent` to `Util.h` and applied it to the calculator.